### DISCUSSION TITLE
problema conbee2

### DISCUSSION THREAD

**Discussion by federico89mi** [indomus/forum#716] [Hardware & Firmware]
Buongiorno,
vi chiedo per favore un opinione in merito a dei problemi che sto avendo con la mia chiavetta conbee2 acquistata qualche mese fa. In poche parole ho iniziato ad utilizzare raspberry con hassio e conbee 2 dopo mesi che avevo lasciato tutto fermo. Avevo fatto dei test mesi fa con hassio e conbee e funzionava tutto.
Ora a distanza di mesi ho reinstallato hassio e tutto solo che ho un problema, sembra che il mio raspberry non legga la chiavetta conbee. In realtà l'ha letta ieri sera e sono riuscito ad inserire il device conbee nelle configurazioni di deconz. Ora invece anche provando riavvii vari e staccando e riattacando la chiavetta più volte non la legge più infatti non si avvia nemmeno deconz. 

Vi allego i log, mi sapete aiutare?
21-09-14 11:11:57 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/008 - None
21-09-14 11:12:07 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/009 - None
21-09-14 11:12:07 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:08 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.STARTUP
21-09-14 11:12:08 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
21-09-14 11:12:09 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:09 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/009 - None
21-09-14 11:12:12 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
21-09-14 11:12:19 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/010 - None
21-09-14 11:12:19 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:21 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:21 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/010 - None
21-09-14 11:12:24 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
21-09-14 11:12:24 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.APPLICATION' starting 0 add-ons
21-09-14 11:12:24 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled
21-09-14 11:12:24 INFO (MainThread) [supervisor.core] Supervisor is up and running
21-09-14 11:12:24 INFO (MainThread) [supervisor.host.info] Updating local host information
21-09-14 11:12:24 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
21-09-14 11:12:24 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
21-09-14 11:12:24 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
21-09-14 11:12:25 INFO (MainThread) [supervisor.host.services] Updating service information
21-09-14 11:12:25 INFO (MainThread) [supervisor.host.network] Updating local network information
21-09-14 11:12:26 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
21-09-14 11:12:26 INFO (MainThread) [supervisor.resolution.module] Create new suggestion SuggestionType.CREATE_FULL_BACKUP - ContextType.SYSTEM / None
21-09-14 11:12:26 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.CORE
21-09-14 11:12:27 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
21-09-14 11:12:27 INFO (MainThread) [supervisor.host.manager] Host information reload completed
21-09-14 11:12:27 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.PLUGIN
21-09-14 11:12:31 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/011 - None
21-09-14 11:12:31 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:33 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:33 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/011 - None
21-09-14 11:12:35 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
21-09-14 11:12:35 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
21-09-14 11:12:35 INFO (MainThread) [supervisor.resolution.check] System checks complete
21-09-14 11:12:35 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
21-09-14 11:12:38 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
21-09-14 11:12:38 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
21-09-14 11:12:38 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
21-09-14 11:12:43 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/012 - None
21-09-14 11:12:43 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:45 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:45 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/012 - None
21-09-14 11:12:48 INFO (SyncWorker_0) [supervisor.docker.addon] Starting Docker add-on homeassistant/aarch64-addon-deconz with version 6.10.0
21-09-14 11:12:55 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/013 - None
21-09-14 11:12:55 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:57 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:12:57 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/013 - None
21-09-14 11:13:07 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/014 - None
21-09-14 11:13:07 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:09 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:09 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/014 - None
21-09-14 11:13:19 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/015 - None
21-09-14 11:13:19 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:21 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:21 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/015 - None
21-09-14 11:13:31 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/016 - None
21-09-14 11:13:31 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:33 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:33 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/016 - None
21-09-14 11:13:43 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/017 - None
21-09-14 11:13:43 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:44 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:44 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/017 - None
21-09-14 11:13:55 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/018 - None
21-09-14 11:13:55 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:57 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:13:57 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/018 - None
21-09-14 11:14:07 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/019 - None
21-09-14 11:14:07 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:10 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:10 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/019 - None
21-09-14 11:14:20 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/020 - None
21-09-14 11:14:20 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:22 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:22 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/020 - None
21-09-14 11:14:32 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/021 - None
21-09-14 11:14:32 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:34 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:34 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/021 - None
21-09-14 11:14:44 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/022 - None
21-09-14 11:14:44 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:46 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:46 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/022 - None
21-09-14 11:14:56 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/023 - None
21-09-14 11:14:56 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:58 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:14:58 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/023 - None
21-09-14 11:15:08 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/024 - None
21-09-14 11:15:08 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:15:10 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:15:10 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/024 - None
21-09-14 11:15:20 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/025 - None
21-09-14 11:15:20 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:15:22 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:15:22 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/bus/usb/001/025 - None
21-09-14 11:15:32 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/bus/usb/001/026 - None
21-09-14 11:15:32 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00
21-09-14 11:15:35 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.REMOVE hardware /dev/ttyACM0 - /dev/serial/by-id/usb-dresden_elektronik_ingenieurtechnik_GmbH_ConBee_II_DE2432357-if00

**Comment by indomus** [comment]
Hai effettuato [l'aggiornamento firmware Crossfade?](https://indomus.it/news/hot-aggiornamento-crossfade-per-deconz-software-e-firmware/)

**Reply by federico89mi** [reply]
in realtà non lo so, ieri deconz mi ha chiesto un aggiornamento del firmware tramite l'interfaccia web phoscon, però non so altro

**Reply by federico89mi** [reply]
avete qualche idea?

**Reply by federico89mi** [reply]
il problema principale mi pare sia che non vede proprio l'hardware negli host, invece ieri me la vedeva

**Reply by indomus** [reply]
Beh, intanto assicurati che deCONZ sia aggiornato e con lui il firmware di ConBee II.

**Reply by federico89mi** [reply]
deconz l'ho installato da zero più volte quindi direi di si. Mentre il conbee non lo so perchè non la legge più come potete vedere dai log

**Reply by KC-inDomus** [reply]
prova ad aggiornare il firmare di conbee usando windows, poi riprova.

**Reply by federico89mi** [reply]
fa la stessa cosa anche su windows, la riconosce per qualche secondo (si sente il bip di windows che riconosce una nuova periferica) e poi si disconnette da sola. Mi sa che si è bruciata, calcolando che l'ho usata si e no 2 volte in 6 mesi non è il massimo. Contatterò amazon per la sostituzione se riesco

**Reply by KC-inDomus** [reply]
è probabile
